src/style.css

```diff
@@ -0,0 +1,62 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  --active: #00acdc;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.item {
+  position: relative;
+  padding: 22.5px 0;
```

Comment:
It's really bad practice to use such 'magic' numbers. You could use display:flex and align-items:center

Reply:
I don't know how to place 'active' underline without paddind